package.json

```diff
@@ -19,19 +19,24 @@
   },
   "homepage": "https://github.com/juasmartinezbelS4N/nodejs_course#readme",
   "devDependencies": {
+    "@types/express": "^4.17.21",
     "@types/jest": "^29.0.0",
     "@types/node": "^18.7.16",
     "@types/supertest": "^2.0.12",
+    "express": "^5.0.0-beta.2",
     "jest": "^29.0.3",
-    "nodemon": "^3.0.3",
+    "joi": "^17.12.2",
+    "logger": "^0.0.1",
+    "nodemon": "^3.1.0",
```

Comment:
`express` and `joi` should not be dev dependencies. They are main dependencies used to run our API, without them you are not able to run it properly in a prod environment.